Source/Core/InputCommon/DynamicInputTextureConfiguration.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <map>
+#include <string>
+
+#include "Common/CommonTypes.h"
+#include "Common/IniFile.h"
+
+namespace InputCommon
+{
+class DynamicInputTextureConfiguration
+{
+public:
+  explicit DynamicInputTextureConfiguration(const std::string& json_file);
```

Comment:
This class should likely default the destructor within the cpp file, given it contains several non-trivial types.